File: Final/untitled.pl
#!/usr/bin/perl
$count = 0;
$max = 100;
$sum = 0;
@nums;
while($count < $max){ 
	$randomint = int(rand(49))-17;
	push(@nums,$randomint*$randomint);
	$sum = $sum+$randomint*$randomint;
	$count = $count+1;
}
$avg = $sum/100;
$count = 0;
$high=0;
while($count<$max){
	if($nums[$count]>$avg){
		$high = $high +1;
	}$count = $count+1;
}
print $high," ", $avg;

File: Final/untitled.py
import random
def getAvg():
	global a
	sum = 0 
	for i in range(0,99):
		rand = int(random.random()*49)-17
		sum = sum + rand*rand
		a.append(rand*rand)
	avg = sum/100
	return avg
def getHigh(avg):
	global a
	high = 0
	for i in range(0,99):
		if(a[i]>avg):
			high = high +1
	return high
a = []
print str(getAvg()) + " " + str(getHigh(getAvg()))